distributed/scheduler.py

```diff
@@ -6828,7 +6842,7 @@ def add_keys(self, worker=None, keys=(), stimulus_id=None):
         ws: WorkerState = self.workers[worker]
         redundant_replicas = []
         for key in keys:
-            ts: TaskState = self.tasks.get(key)
+            ts: TaskState | None = self.tasks.get(key)
```

Comment:
```suggestion
            ts = self.tasks.get(key)
```
SchedulerState is fully annotated